### ISSUE TITLE
Node status: Waiting for network

### ISSUE THREAD

**Issue by thanhnguyennguyen** [Shardeum/shardeum-bug-reporting#464]
##### What is the issue?

- Start node, node status is Standby
- Wait for 10 minutes, Node is moving to `Waiting for network`
- Stuck in `Waiting for network` for a day
Already staked 10 SHM 
<img width="579" alt="image" src="https://github.com/Shardeum/shardeum-bug-reporting/assets/17243442/e67dbd68-7c1b-40dd-b1cb-6ac16680dd2a">

##### How to reproduce the issue? 
-  Just start the node follow docs https://docs.shardeum.org/node/run/validator and I face the issue
### What other resources can you share regarding this issue?

./shell
pm2 list
pm2 log 1

Below is all logs from beginning 

```
init archiver module
init archiver module
initializing join protocol v2
External server running on port 9001...
active: 281, syncing: 18, flag: false
Running isReadyToJoin cycle:34708 publicKey: 04dca6502bdd948a510bcce9f2eb94608ed72302ec63fc10a3ff76abe5c0a6d6
error in startupV2 > attemptJoining: Node not ready to join

Error: Node not ready to join
    at joinNetworkV2 (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:363:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async attemptJoining (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:255:21)
active: 296, syncing: 4, flag: false
Running isReadyToJoin cycle:34834 publicKey: 04dca6502bdd948a510bcce9f2eb94608ed72302ec63fc10a3ff76abe5c0a6d6
queryCertificate { reason: 'Operator certificate has expired', success: false }
error in startupV2 > attemptJoining: Node not ready to join
Stack trace:
Error: Node not ready to join
    at joinNetworkV2 (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:363:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async attemptJoining (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:255:21)
active: 294, syncing: 5, flag: false
Running isReadyToJoin cycle:34835 publicKey: 04dca6502bdd948a510bcce9f2eb94608ed72302ec63fc10a3ff76abe5c0a6d6
queryCertificate { reason: 'Operator certificate has expired', success: false }
error in startupV2 > attemptJoining: Node not ready to join
Stack trace:
Error: Node not ready to join
    at joinNetworkV2 (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:363:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async attemptJoining (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:255:21)
active: 295, syncing: 6, flag: false
Running isReadyToJoin cycle:34836 publicKey: 04dca6502bdd948a510bcce9f2eb94608ed72302ec63fc10a3ff76abe5c0a6d6
queryCertificate { reason: 'Operator certificate has expired', success: false }
error in startupV2 > attemptJoining: Node not ready to join
Stack trace:
Error: Node not ready to join
    at joinNetworkV2 (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:363:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async attemptJoining (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:255:21)
active: 297, syncing: 4, flag: false
Running isReadyToJoin cycle:34837 publicKey: 04dca6502bdd948a510bcce9f2eb94608ed72302ec63fc10a3ff76abe5c0a6d6
error in startupV2 > attemptJoining: Node not ready to join
Stack trace:
Error: Node not ready to join
    at joinNetworkV2 (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:363:15)
    
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async attemptJoining (/usr/src/app/node_modules/@shardus/core/dist/p2p/Self.js:255:21)
active: 299, syncing: 2, flag: false
Running isReadyToJoin cycle:34838 publicKey: 04dca6502bdd948a510bcce9f2eb94608ed72302ec63fc10a3ff76abe5c0a6d6
queryCertificate {
  signedStakeCert: {
    certExp: 1702347087665,
    nominator: '0x9059f758e2d7ab314407cf6ae821b024e538c792',
    nominee: '04dca6502bdd948a510bcce9f2eb94608ed72302ec63fc10a3ff76abe5c0a6d6',
    signs: [ [Object], [Object], [Object], [Object], [Object] ],
    stake: '8ac7230489e80000'
  },
  success: true
}
stakeCert received. remainingValidTime: 1738731 expiredPercent: -1.897885, isNewCertExpiringSoon: false
valid cert, isReadyToJoin = true  {
  certExp: 1702347087665,
  nominator: '0x9059f758e2d7ab314407cf6ae821b024e538c792',
  nominee: '04dca6502bdd948a510bcce9f2eb94608ed72302ec63fc10a3ff76abe5c0a6d6',
  signs: [
    {
      owner: '8a4d4c916b1f6b56f1be57810aca27e18df51029736d0bcbdbe821de48a1459f',
      sig: 'a5bae6ef10799250875c016daf2a287977ee7783f1dbbf4584946eb122e402366421ce40fffd7639cb41c3fc62fbe749755107ecc10894ae92eae393886f58075428b6bcefd718afccf27c96150a3ad4c3efc1b11175d132b7ed77cb678bfc4e'
    },
    {
      owner: '624ed65c80a06e27c782e8f99ce9beacc6942649585422dba92af174a02413c8',
      sig: '36b6802461838736f8131ab1792a1729e267402eed0a5c283c90464314b2a9075e1e0a9184c2b72736fbc3ab3b4a7dddcee3d831a02e2258068de23da1c9fe0c5428b6bcefd718afccf27c96150a3ad4c3efc1b11175d132b7ed77cb678bfc4e'
    },
    {
      owner: 'dcdb41570da5e482d521d3277d73ade75f2c69a7afddff55996670ff278224cb',
      sig: '8660addfa643722595c058ec34c95ec5ced760bf57526106354cd4188c9a75f631dc74615451b6b4d0c6c197ec1123852d26cd19ede2b405132a5b33ac0bf8055428b6bcefd718afccf27c96150a3ad4c3efc1b11175d132b7ed77cb678bfc4e'
    },
    {
      owner: '643a87f22e3c6fe954194d8a716f1b5438adc5f3f330431fe90d932ace847937',
      sig: '14009992f3c329e9da830e6d2e8162c4de3b6e23c4f822ba26e258e88ea849decb9af8fcd853b041ad0a36b6afd89a3a50ccfc2e4471703a406436c96c3947095428b6bcefd718afccf27c96150a3ad4c3efc1b11175d132b7ed77cb678bfc4e'
    },
    {
      owner: '8560816cc95fede59dfb7086826b3534c3449b81252abab2586a33c9adcdb1c1',
      sig: '77bc31139f4bb90dd6dacc544db2d5d479601eadc2156878f0e849d1cd8f6c87a3a26a38033c15ac673be54a3937fb5096e7e15be41dd00abeb220e7d8a63c0e5428b6bcefd718afccf27c96150a3ad4c3efc1b11175d132b7ed77cb678bfc4e'
    }
  ],
  stake: '8ac7230489e80000'
}
error in startupV2 > attemptJoining: Response code 400 (Bad Request)
Stack trace:
HTTPError: Response code 400 (Bad Request)
    at EventEmitter.<anonymous> (/usr/src/app/node_modules/got/source/as-promise.js:74:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    
```

Stuck here forever

**Comment by thanhnguyennguyen** [comment]
this is the last log, no more log for hours
status moving from `Waiting for network` to `Standby` and stuck there for hours

```
error in startupV2 > attemptJoining: Response code 400 (Bad Request)
Stack trace:
HTTPError: Response code 400 (Bad Request)
    at EventEmitter.<anonymous> (/usr/src/app/node_modules/got/source/as-promise.js:74:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    ```

**Comment by thanhnguyennguyen** [comment]
after 4 hours, status moving from Standby to Waiting for network

**Comment by swarna1101** [comment]
update to the latest node its solved in latest version 1.9.5.
